modules/claims_api/lib/bd/bd.rb

```diff
@@ -172,11 +175,11 @@ def client
 
     def build_body(options = {})
       data = {
-        systemName: options.fetch(:system_name, 'VA.gov'),
+        systemName: options[:system_name].nil? ? 'VA.gov' : options[:system_name],
         docType: options[:doc_type],
         claimId: options[:claim_id],
         fileName: options[:file_name],
-        trackedItemIds: options.fetch(:tracked_item_ids, [])
```

Comment:
nit: why are we reinventing [`fetch`](https://apidock.com/ruby/Hash/fetch)?

Reply:
@tycol7 I believe we want 'VA.gov' if the systemName is null, and fetch doesn't seem to be working as expected here. @klwatanabe & @FonzMP is that correct, we want the default to be 'VA.gov'?
![Screenshot 2024-07-29 at 10 44 20 AM](https://github.com/user-attachments/assets/5b1bbb77-b527-4915-9d84-591602c06ca6)